jproperties-checks/src/main/java/org/sonar/jproperties/checks/CheckUtils.java

```diff
@@ -34,4 +39,10 @@ public static String paramsErrorMessage(Class clazz, String message) {
       + message;
   }
 
+  public static final Set<String> LOCALES = Arrays.asList(DateFormat.getAvailableLocales())
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this variable to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1213)